docs/reference/extensions/django-framework.rst

```diff
@@ -43,6 +50,26 @@ application. In the following example we use it to specify ``libpq-dev``:
         # list required packages or slices for your Django application below.
         - libpq-dev
 
+.. _django-async-gunicorn-workers:
+
+Using asynchronous Gunicorn workers
+===================================
+
+If you want to use asynchronous workers, you have to add the ``gevent`` package
+to the ``requirements.txt`` file. Rockcraft automatically detects this and
+updates the Pebble plan to use the asynchronous workers. If you have ``gevent``
+installed in your rock but decided to use ``sync`` workers instead you can do
+so by using the ``--args`` parameter of ``docker run``:
+
+.. code-block:: bash
+
+   docker run --name django-container -d -p 8000:8000 django-image:1.0 \
+   --args django sync
+
+.. note::
+    The Django extension is compatible with the ``bare``, ``ubuntu@22.04`` and
+    ``ubuntu@24.04`` bases.
```

Comment:
This looks like it was misplaced. Can we move this to the introduction, like in the other page?